Walkthrough Guide/06 Functions and Cognitive Services/README.md

```diff
@@ -133,7 +133,7 @@ Both `Stream` objects get passed to the Function. The rest of the code just conn
 
 ### 2.3 Integrate with Storage, Cosmos DB and Cognitive Services
 
-Of course, all these Trigger, Input and Output [Bindings](https://docs.microsoft.com/en-us/azure/azure-functions/functions-triggers-bindings) have to be configured. As we might be already used to from the App Service, this configuration is done via Environment Variables. Each Azure Function has a `local.settings.json` file that sets Connection Strings to the used services.
+Of course, all these Triggers, Inputs and Outputs [Bindings](https://docs.microsoft.com/en-us/azure/azure-functions/functions-triggers-bindings) have to be configured. As we might be already used to from the App Service, this configuration is done via Environment Variables. Each Azure Function has a `local.settings.json` file that sets Connection Strings to the used services.
```

Comment:
Is it really called "Trigger**s** Bindings" instead of a "Trigger Bindings"? Asking as a non-native speaker...